docs/source/operations/overview.rst

```diff
@@ -23,6 +23,16 @@ For each component section below, configuration options relates to the component
 
 .. _cert-manager: https://cert-manager.io
 
+.. note::
+
+    We provide full support on the following OS:
```

Comment:
I would say we provide support to every major OS (Ubuntu, MacOS, Windows). The difference is more on how deep we test them.

So we could rephrase:

> We provide full support on the following OS

By something like "the following OS version are deeply tested".


And for windows saying that only Windows Server 2022 is tested and only on subprocess modes. Howevers users should feel free to try other Windows version, and reach out on Slack if something goes wrong.

Reply:
We can't really provide much support on Windows due to not having Windows machines available though.

It's not a total blocker but if someone asks an Apple question we can answer a lot more rapidly (and probably better) than a Windows question

Reply:
From my understanding, the idea here was to limit our promises on the support we can provide.